CWE/events/Austria - Historic.txt

```diff
@@ -0,0 +1,698 @@
+#1949 Election
+country_event = {
+	id = 8213130
+	title = "$YEAR$ Austrian Parliamentary Election"
+	desc = EVTDESC8213130
+	picture = "aus_early_election"
+	
+	fire_only_once = yes
+
+	trigger = {
+		OR = { tag = AUS tag = ROA }
+		year = 1949
+		NOT = { year = 1953 }
+		
+		government = democracy
+		war = no
+		OR = { is_vassal = no is_vassal = USA }
+		
+		OR = { ruling_party_ideology = socialist ruling_party_ideology = conservative ruling_party_ideology = liberal }
+		
+		#Some form of parliamentary government
+		OR = {
+			has_country_flag = semi_presidential_system_republic
+			has_country_flag = parliamentary_system_republic
+		}
+	}
+
+	mean_time_to_happen = { months = 6 }
+
+	option = {
+		name = "The Austrian People's Party retains their grip on politics."
+
+		ruling_party_ideology = conservative
+
+		any_pop = { 
+			ideology = { factor = 0.2 value = conservative }
+		}
+
+		ai_chance = { 
+			factor = 0.8
+			modifier = {
+				factor = 6
+				pop_majority_ideology = conservative
+			}
+			modifier = {
+				factor = 0.25
+				NOT = { pop_majority_ideology = conservative }
+			}
+		}
+	}
+
+	option = {
+		name = "Social Democrats manage to gain majority over their coalition."
+
+		ruling_party_ideology = socialist
+
+		any_pop = { 
+			ideology = { factor = 0.2 value = socialist }

```

Comment:
Effects adding socialist ideology should exclude rich strata.

Reply:
Alright, that should all be fixed now.